store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreSchemaTest.java

```diff
@@ -70,95 +94,210 @@ public void setUp() throws Exception {
         fStore = new FederatedStore();
         fStore.initialise(TEST_FED_STORE, null, FEDERATED_PROPERTIES);
 
-        fStore.setGraphLibrary(LIBRARY);
-
         testUser = testUser();
         testContext = new Context(testUser);
     }
 
     @Test
     public void shouldBeAbleToAddGraphsWithSchemaCollisions() throws Exception {
-        LIBRARY.addProperties(ACC_PROP, PROPERTIES);
-        fStore.setGraphLibrary(LIBRARY);
+        // Given
+        addGroupCollisionGraphs();
 
-        String aSchema1ID = "aSchema";
         final Schema aSchema = new Schema.Builder()
```

Comment:
This schema creation looks like its being used multiple times, can you set it as a Constant.